pro/src/components/IndividualOffer/StocksEventCreation/StocksCalendar/StocksCalendarTable/StocksCalendarTable.tsx

```diff
@@ -0,0 +1,129 @@
+import { format } from 'date-fns'
+
+import {
+  GetIndividualOfferResponseModel,
+  GetOfferStockResponseModel,
+} from 'apiClient/v1'
+import { getPriceCategoryName } from 'components/IndividualOffer/StocksEventEdition/getPriceCategoryOptions'
+import fullTrashIcon from 'icons/full-trash.svg'
+import { Checkbox } from 'ui-kit/formV2/Checkbox/Checkbox'
+import { ListIconButton } from 'ui-kit/ListIconButton/ListIconButton'
+
+import styles from './StocksCalendarTable.module.scss'
+
+export function StocksCalendarTable({
+  stocks,
+  offer,
+  onDeleteStocks,
+  checkedStocks,
+  updateCheckedStocks,
+}: {
+  stocks: GetOfferStockResponseModel[]
+  offer: GetIndividualOfferResponseModel
+  onDeleteStocks: (id: number[]) => void
+  checkedStocks: Set<number>
+  updateCheckedStocks: (newStocks: Set<number>) => void
+}) {
+  function handleStockCheckboxClicked(stock: GetOfferStockResponseModel) {
```

Comment:
détail : on à l'air de n'utiliser que l'id dans la fonction
```suggestion
  function handleStockCheckboxClicked(stockId: int) {
```